platform/innovium/docker-syncd-invm-rpc.mk

```diff
@@ -2,11 +2,11 @@
 
 DOCKER_SYNCD_INVM_RPC = docker-syncd-invm-rpc.gz
 $(DOCKER_SYNCD_INVM_RPC)_PATH = $(PLATFORM_PATH)/docker-syncd-invm-rpc
-$(DOCKER_SYNCD_INVM_RPC)_DEPENDS += $(SYNCD_RPC) $(LIBTHRIFT) $(INVM_LIBSAI) $(PTF) $(INVM_SHELL)
+$(DOCKER_SYNCD_INVM_RPC)_DEPENDS += $(SYNCD_RPC) $(INVM_LIBSAI) $(INVM_SHELL)
+$(DOCKER_SYNCD_INVM_RPC)_PYTHON_WHEELS += $(PTF_PY3)
 $(DOCKER_SYNCD_INVM_RPC)_FILES += $(SUPERVISOR_PROC_EXIT_LISTENER_SCRIPT)
 $(DOCKER_SYNCD_INVM_RPC)_LOAD_DOCKERS += $(DOCKER_SYNCD_BASE)
 SONIC_DOCKER_IMAGES += $(DOCKER_SYNCD_INVM_RPC)
-SONIC_BUSTER_DOCKERS += $(DOCKER_SYNCD_INVM_RPC)
```

Comment:
Can you keep this line, but have it be `SONIC_BOOKWORM_DOCKERS`? It won't have any effect now, but it will make the upgrade to Debian Trixie in the future a bit easier.

Reply:
thanks, I have kept the line with SONIC_BOOKWORM_DOCKERS in updated patch